pkg/jsonlog/jsonlogbytes.go

```diff
@@ -5,10 +5,10 @@ import (
 	"unicode/utf8"
 )
 
-// JSONLogBytes is based on JSONLog.
+// JSONLogbytes is based on JSONLog.
 // It allows marshalling JSONLog from Log as []byte
 // and an already marshalled Created timestamp.
-type JSONLogBytes struct {
+type JSONLogbytes struct {
```

Comment:
@aaronlehmann  golint will show this warnings for JOSNLogBytes `jsonlogbytes.go:11:6: type name will be used as jsonlog.JSONLogBytes by other packages, and that stutters; consider calling this Bytes, so I change it to `Logbytes`
`